scripts/enrich_rulings_ai.py

```
@@ -12,96 +14,146 @@
 from bs4 import BeautifulSoup  # For stripping HTML if needed from original_html_snippet
 
 logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
 # DEFAULT_SOURCE_CARD_CODE_EXTERNAL is now in constants.py
 
+api_key = os.environ.get("OPENAI_API_KEY")
+if not api_key:
+    logger.error("OPENAI_API_KEY environment variable not set. Please set it before running this script.")
```

Ruff (0.11.9)

Line too long (106 > 100)

(E501)